integration_tests/pages/assess/taskListPage.ts

```diff
@@ -18,4 +18,15 @@ export default class TaskListPage extends Page {
   shouldNotShowSection = (section: string): void => {
     cy.get('.app-task-list').should('not.contain', section)
   }
+
```

Comment:
Would it be worth combining these two pages taskListPage.ts into one? they are basically the same other than 'Apply/Assess' in the constructor and the `visit` method